cli/tests/test_config_command.rs

```diff
@@ -945,6 +945,67 @@ fn test_config_edit_user_new_file() {
     );
 }
 
+// TODO: remove after deprecation period in 0.35
+// NOTE: macOS-only test
+#[test]
+fn test_config_edit_user_deprecated_file() {
+    if !cfg!(target_os = "macos") {
+        return;
+    }
+    let mut test_env = TestEnvironment::default();
+    let mut user_config_path = test_env.home_dir().join("Library/Application Support/jj");
+    std::fs::create_dir_all(&user_config_path).unwrap();
+    user_config_path.push("config.toml");
+
+    std::fs::write(
+        &user_config_path,
+        indoc! {b"
+        [foo]
+        bar = 'Make sure I can pick this up'
+        "},
+    )
+    .unwrap();
+
+    test_env.set_up_fake_editor(); // set $EDIT_SCRIPT, but added configuration is ignored
+    test_env.add_env_var("EDITOR", fake_editor_path());
+
```

Comment:
nit: maybe we don't need the editor in this test?

Reply:
I think you're right.